ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-ref.yaml

```diff
@@ -8,5 +8,12 @@ ref:
       memory: 600Mi
     limits:
       memory: 2Gi
+  env:
+  - name: RUN_UPGRADE_TEST
+    default: "false"
+    documentation: If the value is equal to "true", run only the upgrade tests
+  dependencies:
+  - name: "release:latest"
+    env: OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE
```

Comment:
It would be better probably to use `OPENSHIFT_UPGRADE_RELEASE_IMAGE`, see changes currently in progress on https://github.com/openshift/release/pull/12959

Reply:
Fixed